_episodes/11-snakemake-intro.md

```diff
@@ -154,6 +154,15 @@ isles	3822	2460	1.55
 ```
 {: .output}
 
+> ## Zipf's Law
+>
+> is an [empirical law](https://en.wikipedia.org/wiki/Empirical_law) formulated using [mathematical statistics](https://en.wikipedia.org/wiki/Mathematical_statistics) that refers to the fact that many types of data studied in the physical and social sciences can be approximated with a Zipfian distribution, one of a family of related discrete [power law](https://en.wikipedia.org/wiki/Power_law) [probability distributions](https://en.wikipedia.org/wiki/Probability_distribution).
```

Comment:
Repeat "Zipf's Law" to make the first clause a sentence.

Reply:
see 7e8b0b8